tests/JustSaying.UnitTests/Messaging/Channels/ErrorHandlingTests.cs

```diff
@@ -52,7 +53,13 @@ IEnumerable<Message> GetMessages()
             .WithDefaultConcurrencyLimit(8);
         var settings = new Dictionary<string, SubscriptionGroupConfigBuilder>
         {
-            { "test", new SubscriptionGroupConfigBuilder("test").AddQueues(queues) },
+            {
+                "test", new SubscriptionGroupConfigBuilder("test").AddQueue(new SqsSource
+                {
+                    SqsQueue = queue,
+                    MessageConverter = new ReceivedMessageConverter(new NewtonsoftMessageBodySerializer<SimpleMessage>(), new MessageCompressionRegistry(), false)
```

Comment:
Would using STJ instead of Newtonsoft in all these tests (except the ones explicitly testing that implementation) speed things up further?